ko/extend_your_application/README.md

```diff
@@ -105,14 +103,16 @@ Post.objects.get(pk=pk)
 
 좋아요. 이제 `views.py`파일에 새로운 *뷰*를 추가합시다!
 
-`blog/views.py`파일을 열고 아래 코드를 추가하세요.
+`blog/urls.py`파일에서 `views.post_detail`라는 뷰를 `post_detail`이라 이름을 붙이도록 URL 법칙을 만들었어요. 이는 장고가 `post_detail`이라는 이름을 해석할 때, `blog/views.py`파일 내부의 `post_detail`이라는 뷰 함수로 이해하도록 해줍니다.
+
+`blog/views.py`파일을 열고, `from`으로 시작하는 행에서 다음과 같이 코드를 추가해주세요:
```

Comment:
[Add translation] Original Text:
`In blog/urls.py we created a URL rule named post_detail that refers to a view called views.post_detail. This means that Django will be expecting a view function called post_detail inside blog/views.py.`

`We should open blog/views.py in the code editor and add the following code near the other from lines:`